test/__snapshots__/NFTDescriptor.spec.ts.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`NFTDescriptor #constructTokenURI gas 1`] = `1632887`;
+exports[`NFTDescriptor #constructTokenURI gas 1`] = `1632890`;
```

Comment:
@intrinsic-eng - is there a reason to change the NFT descriptor?

I don't mind changing it, just wondering if there is a reason.

Reply:
The change in that snapshot file is due to a change in the output from this snapshot test of the gas cost. https://github.com/Intrinsic-network/periphery/blob/24ce5ca712a77a651a9b6cd0a9429378d4b9bd14/test/NFTDescriptor.spec.ts#L311

"It is due to the change I made to quoteTokenSymbol value from 'UNI' to 'INT'